test/e2e/example_k8petstore.go

```diff
@@ -150,7 +160,7 @@ T:
 	// We should have exceeded the finalTransactionsExpected num of transactions.
 	// If this fails, but there are transactions being created, we may need to recalibrate
 	// the finalTransactionsExpected value - or else - your cluster is broken/slow !
-	Ω(totalTransactions).Should(BeNumerically(">", finalTransactionsExpected))
+	Expect(totalTransactions).Should(BeNumerically(">", finalTransactionsExpected))
```

Comment:
According to their [doc](https://onsi.github.io/gomega/#making-assertions), you need to change "Should" to "To"